plugins/gdtf/gdtf-attributes.js

```diff
@@ -951,17 +974,19 @@ const gdtfAttributes = {
     oflProperty: `shakeSpeed`,
     defaultPhysicalEntity: `Frequency`,
     beforePhysicalPropertyHook(capability, gdtfCapability) {
```

Comment:
Capability type should be `WheelShake`, but inheriting form `Gobo1` will result in `WheelSlot`.